packages/addons/browser/chrome/source/default.py

```diff
@@ -74,28 +87,31 @@ def getAudioDevice():
                    })))['result']['value']
 
 def getAudioDeviceType(dev):
-  if dev.startswith("ALSA:"):
-    return "ALSA"
-  if dev.startswith("PULSE:"):
-    return "PULSE"
-  return None
+  if dev:
+    if dev.startswith("ALSA:") or dev.startswith('plughw:'):
+      return "ALSA"
+    elif dev.startswith("PULSE:"):
+      return "PULSE"
```

Comment:
In the past every custom device was an alsa one (had to open a new comment for the suggestion):
```suggestion
    if dev.startswith("ALSA:"):
      return "ALSA"
    elif dev.startswith("PULSE:"):
      return "PULSE"
    else:
      # Backward compatibility of device string
      return "ALSA"
```

Reply:
Got it now, thanks. Will provide the change in an update to this PR.

Reply:
OK, updated the PR according to your suggestion @mglae